mongodb/6.3/modules/ROOT/pages/mongodb-connector-troubleshooting.adoc

```diff
@@ -0,0 +1,45 @@
+= Troubleshooting MongoDB Connector 6.3 - Mule 4
+
+To troubleshoot Anypoint Connector for MongoDB (MongoDB Connector), become familiar with the information about interpreting commonly thrown exception messages.
+
+== Understand Commonly Thrown Exceptions
+
+Here is a list of commonly thrown exception messages and how to interpret them:
+
+* MONGO:CONNECTIVITY
+
+ Either a connection could not be established or there is a connectivity error.
```

Comment:
General comment (can be looked at in the future) - some of these descriptions are in the present tense and some are in the past tense.

Reply:
The text for this description is inconsistent:
"could not be established" goes with there was a "connectivity error"